### DISCUSSION TITLE
Port to Mac and/or Linux

### DISCUSSION THREAD

**Discussion by milos-pajic-doodle** [DevToys-app/DevToys#163] [Ideas]
Hey, is there any plan on porting this to Mac OS or to Linux distro?
Thanks, 
M

**Comment by antonpkazakov** [comment]
It can probably even be a PWA that is extractible via Chrome into a 'standalone' app.

**Comment by assoft** [comment]
There should definitely be cross-platform support...

**Comment by riccamgom** [comment]
Maybe with wine or PlayOnLinux could be easy to give it support on Linux

**Comment by raphaelsamyfacily** [comment]
Looks like this is no longer needed, only the ux is a little worse https://github.com/gchq/CyberChef

**Comment by gumbarros** [comment]
@milos-pajic-doodle I'm developing a Linux port
https://www.github.com/gumbarros/devtoysflutter
If you're a Mac user there is also https://github.com/ObuchiYuki/DevToysMac

**Comment by flaunder007** [comment]
The Linux version is still an up-to-date request. Can you make an online version so that it can be used at least as a PWA?
@gumbarros thank's, but you dont have such tools as DevToys. However, this is still the best solution.

**Reply by veler** [reply]
Hi,
DevToys on Mac and Linux are a work in progress. It is currently in the branch 'dev/2.0' of the repository. We're hoping for a preview version early 2024.

**Reply by ABAG603** [reply]
Hi @veler . How does it look with Linux/Mac version? Was it pushed back?

**Reply by veler** [reply]
It's looking good! Will be release in a few weeks. We're finalizing details and are preparing for release.

**Reply by ABAG603** [reply]
@veler I was very happy to see v2 released today. Any plans to have an "official" Homebrew Cask for it?

**Reply by veler** [reply]
> @veler I was very happy to see v2 released today. Any plans to have an "official" Homebrew Cask for it?

Thanks!Homebrew is a work in progress. I'm hoping to get it done today : https://github.com/Homebrew/homebrew-cask/pull/175585
